.gitattributes

```diff
@@ -0,0 +1,2 @@
+demos/talk_paper_demos/Parslfest_2025/Parsl_Testbook_Speedup.deck filter=lfs diff=lfs merge=lfs -text
+demos/talk_paper_demos/Parslfest_2025/Parsl_Testbook_Speedup.pptx filter=lfs diff=lfs merge=lfs -text
```

Comment:
I don't see these files in the branch.

Reply:
These are stored in git-lfs did you do  

`git lfs fetch --all && git lfs checkout  --all `

after cloning the repo?